Short: erq: object for reply destructed
From: Holger@Wunderland
Date: 990217
Type: Bug
State: Done - fixed in 3.2.9-dev.325/3.3.90

The driver seems to 'bug' when an object is destructed for which a reply
from the erq is expected. Investigate this.

---------- Analysis ----------

According to the sources, the secure_call_lambda() will throw an error
in this case, which calls master::runtime() for notification,but in
effect is caught.
